package/main/daml/ContingentClaims.Valuation.V1/daml.yaml

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-sdk-version: 2.10.0-snapshot.20250120.13164.0.vd2e3ddc6
+sdk-version: 2.10.0
 name: contingent-claims-valuation-v1
 source: daml
-version: 0.99.0.20250121.1
+version: 1.0.0
```

Comment:
Mmm, this is still an experimental package so I think it would make sense keep a `0.x` version.

Not sure why it was `0.99` beforehand, I'd just bump the minor version from the last release.